hoomd/md/data/local_access.py

```diff
@@ -35,3 +35,35 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self._force_obj._in_context_manager = False
         self._exit()
+
+
+class _NeighborListLocalAccessBase(hoomd.data.local_access._LocalAccess):
+    __slots__ = ('_entered', '_accessed_fields', '_cpp_obj', '_nlist_obj')
+
+    # This should work without having to comment out the @abstractmethod
+    # decorator, but it just doesn't... Once commented out, it is easily able to
+    # find the _cpp_cls attribute.
+    @property
+    # @abstractmethod  # This has to be commented out for the code to work
+    def _cpp_class(self):
+        pass
+
```

Comment:
This doesn't work as is because the property name is wrong.
```suggestion
    @property
    @abstractmethod
    def _cpp_cls(self):
        pass

```

Reply:
🤦